### ISSUE TITLE
Season Long Stats

### ISSUE THREAD

**Issue by JamesPHawk** [cwendt94/espn-api#599]
### Sport

Basketball

### Summary

Basically it would return the season-long stats under /league/standings.

**Comment by cwendt94** [comment]
Let me look into where that data is being grabbed from!

**Comment by cwendt94** [comment]
I found that the Team class is already saving this data in `stats` field. For each team it will have their season long stats!

**Comment by adamhrogers** [comment]
@cwendt94 Would it be possible to get season long PF and PA for a team?

**Comment by cwendt94** [comment]
@adamhrogers yeah I can add that to the team class. Thanks for pointing that out, I thought it already had it.
